app/services/case_linking_service.rb

```diff
@@ -37,24 +37,42 @@ def call
   private
 
   def validate_params
+    validate_case_number_present
+    validate_linked_case_not_this_case
+    validate_linked_case_exists
+    validate_sar_case_linking
+  end
+
+  def validate_sar_case_linking
+    if @case.is_a?(Case::SAR)
```

Comment:
I think we should be doing this check in the `SARPolicy`, otherwise we'll have 3 places to configure this: `BasePolicy`, `User` and here.

Reply:
Not sure what you mean here.